app/controllers/new-signup.js

```diff
@@ -21,12 +21,18 @@ export default class NewSignUpController extends Controller {
   SECOND_STEP = NEW_SIGNUP_STEPS[1];
   THIRD_STEP = NEW_SIGNUP_STEPS[2];
   FOURTH_STEP = NEW_SIGNUP_STEPS[3];
-  LAST_STEP = NEW_SIGNUP_STEPS[4];
+  FIFTH_STEP = NEW_SIGNUP_STEPS[4];
+  LAST_STEP = NEW_SIGNUP_STEPS[5];
+
+  get isDevMode() {
+    return this.dev || false;
+  }
```

Comment:
A service has been created for it in the PR, Why are we not using that here?

Reply:
Done with changes...can you please check my latest commit